### PR TITLE
added country config

### PR DESCRIPTION
**Pull request by zacel**
my local env is rewriting the locale files whenever I start the server to test but everything is setup and looks good

### CODE REVIEW

File: app/views/pages/report.ejs [hunk 1]
```diff
@@ -138,7 +138,9 @@
             <a target="_blank" href="<%= urls.profile %>/<%= passcode %>"><%= baseUrl %><%= urls.profile %>/<%= passcode %></a>.
           </div>
         <% } else { %>
-          <p class="slogan"><%= __("Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ") %></p>
+          <p class="slogan">
+            <%= __('Are you from ') + __(locals.country) + "? "%>
```
Comment (adriaandotcom) [inline app/views/pages/report.ejs]: I would remove the spacing from the translation and add it in the HTML.

File: app/locales/nl.json [hunk 2]
```diff
@@ -135,5 +135,10 @@
   "Clarifications data basis": "Verduidelijkingen gegevensbasis",
   "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:": "Vragen over nauw contact zijn op 16 maart toegevoegd op. 10:30 p.m .. Sommige symptomen maakten vanaf het begin geen deel uit van het formulier:",
   "Symptom": "Symptoom",
-  "Added to form": "Toegevoegd aan formulier"
+  "Added to form": "Toegevoegd aan formulier",
+  "Are you from ":"Kom je uit ",
+  "Norway":"Noorwegen",
+	"Netherlands":"Nederland",
```
Comment (adriaandotcom) [inline app/locales/nl.json]: Oops, tabs here!

**Review by adriaandotcom — CHANGES_REQUESTED** [review]
Two little things

**Comment by zacel** [comment]
Ok will fix these things now

**Comment by fossecode** [comment]
https://github.com/mashpie/i18n-node/issues/266

> Root cause could be any error in you scripts and/or json files. Please ensure that you json files are valid to strict json specs, like using " and not ', don't have trailing commas or comments.

> One possible scenario might be: A wrong typed json file get's read by i18n. An empty locale object get's initiated due to json-parsing-errors and as soon any i18n.__() get invoked i18n writes back a new json file but with freshly initiated object containing keys only.

> The config setting will safe you from destroying your data. But i18n might not work until you fix the root cause of that error.

**Comment by fossecode** [comment]
We could consider setting the following in the i18n config:
```
updateFiles: false
```

**Review by fossecode — APPROVED** [review]

**Comment by zacel** [comment]
Ok thank you fossecode. Maybe it was the trailing commas. My mistake I thought we were supposed to do that still as it was said to do yesterday

**Review by adriaandotcom — APPROVED** [review]

**Comment by adriaandotcom** [comment]
```
updateFiles: false
```

would be great I think

**Comment by mashpie** [comment]
Hi there! Author of `i18n` here... if you need any help with implementation, just drop me a line! 

`updateFiles: false` will be default in an upcoming release

**Pull request merged**